core-blocks/index.js

```diff
@@ -77,11 +78,15 @@ export const registerCoreBlocks = () => {
 		table,
 		textColumns,
 		verse,
-		video,
 	].forEach( ( { name, settings } ) => {
 		registerBlockType( name, settings );
 	} );
 
+	[ video ].forEach( ( { name, definition, implementation } ) => {
+		dispatch( 'core/blocks' ).addBlockTypes( { name, ...definition } );
```

Comment:
I think this is not acceptable solution because it will break all plugins that depend on the following filter:

```js
settings = applyFilters( 'blocks.registerBlockType', settings, name );
```

Reply:
In addition this code bypass all validations we have in `registerBlockType` function.

Reply:
Good points. 

> The filter 

How to approach this? should we just leave the video block as is for now and refactor all the blocks at once once we create the alternative filters and move things to the server? I'm trying to avoid telling people to rewrite their code twice and only introduce deprecations once we have everything.

> In addition this code bypass all validations we have in registerBlockType function.

I guess we should split validation between the two actions. I'll give it a shot

Reply:
Two times yes :)

It was good for illustration how this can work, but we still need those wrapping API functions to make sure everything is valid.